libs/forecasting/src/lib/ForecastingDashboard/Controls/TransformationsTable.tsx

```diff
@@ -124,24 +128,37 @@ export class TransformationsTable extends React.Component<
     );
   }
 
-  public calculateUpdatedTable(): ITransformationRow[] {
-    const rows: ITransformationRow[] = [
-      ...this.props.transformations.entries()
-    ].map(([transformationName, transformation], index) => {
-      const method = `${transformation.feature.text} ${
-        transformation.operation.displayName
-      } ${
-        isMultiplicationOrDivision(transformation.operation)
-          ? localization.Forecasting.TransformationTable
-              .divisionAndMultiplicationBy
-          : ""
-      }${transformation.value.toString()}`;
-      return {
-        key: index.toString(),
-        method,
-        transformationName
-      };
-    });
+  public calculateUpdatedTable(
+    jointDataset: JointDataset
+  ): ITransformationRow[] {
+    const rows: ITransformationRow[] = [...this.props.transformations.entries()]
+      .map(([transformationName, transformation], index) => {
+        let value: string | number = transformation.value;
+        if (jointDataset.metaDict) {
+          const featureMeta = jointDataset.metaDict[transformation.feature.key];
+          if (
+            featureMeta.sortedCategoricalValues &&
+            (featureMeta.isCategorical || featureMeta.treatAsCategorical)
+          ) {
+            value = featureMeta.sortedCategoricalValues[value];
+          }
+          const method = `${transformation.feature.text} ${
```

Comment:
could you comment a sample of how what the final data would look like here

Reply:
Great idea! Adding that.